snapshot/cmd/snapshot-controller/snapshot-controller.go

```diff
@@ -134,7 +134,7 @@ func buildVolumePlugins() {
 				glog.Infof("Register cloudprovider %s", cinder.GetPluginName())
 			}
 		} else {
-			glog.Warningf("failed to initialize cloudprovider: %v, supported cloudproviders are %#v", err, cloudprovider.CloudProviders())
+			glog.Fatalf("failed to initialize cloudprovider: %v, supported cloudproviders are %#v", err, cloudprovider.CloudProviders())
```

Comment:
If you get an x509 auth declined because of the missing ca-certs the application will continue to run claiming you have not provided enough arguments to cinder or some such nonsense.  If applications cannot perform their function they should die IMO.